src/traces/surface/attributes.js

```diff
@@ -182,6 +206,20 @@ module.exports = {
             min: 0.00,
             max: 5.00,
             dflt: 0.2
+        },
+        vertexnormalsepsilon: {
```

Comment:
@etpinard it's quite specific to to the epsilon error terms in the normals calculation and these names would be suggestive of passing on the normals directly.